src/main/scala/nl.knaw.dans.easy.deposit/EasyDepositApiApp.scala

```diff
@@ -293,10 +293,13 @@ class EasyDepositApiApp(configuration: Configuration) extends DebugEnhancedLoggi
     else Success(())
   }
 
-  private def contentTypeAnythingButZip(contentType: Option[String]): Try[Unit] = {
-    if (contentType.exists(str => str.trim.nonEmpty && !str.trim.matches(contentTypeZipPattern)))
-      Success(())
-    else Failure(InvalidContentTypeException(contentType, "must not be a zip."))
+  private def contentTypeAnythingBut(contentType: Option[String]): Try[Unit] = {
+    if (contentType.map(_.trim.toLowerCase).exists(str =>
```

Comment:
would a pattern match make this better readable than an if-else? up to you to decide!

```scala
contentType.map(_.trim.toLowerCase) match {
  case Some(str) if str.nonEmpty
    && !str.startsWith("multipart")
    && !str.matches(contentTypeZipPattern) => Success(())
  case _ => Failure(InvalidContentTypeException(contentType, "must not be application/zip nor start with multipart."))
}
```